docs/docs/code/python/README.md

```diff
@@ -261,6 +288,40 @@ raise NameError('Something happened that should not. Exiting early.')
 
 All exceptions from your Python code will appear in the **logs** area of the results.
 
+## Ending a workflow early
```

Comment:
The formatting on this section looks off, and that's usually a signal that you forgot to close a code block somewhere. Seems like it starts in the **Sending data downstream to other steps** section. Could just be a GitHub rendering issue but wanted to call it out.